LATEX.g4

```diff
@@ -442,33 +478,34 @@ func_normal_functions_single_arg:
     | FUNC_ARCSINH | FUNC_ARCCOSH | FUNC_ARCTANH
     | FUNC_FLOOR | FUNC_CEIL;
 
-func_normal_functions_multi_arg:
+func_cmd_multi_arg:
     FUNC_GCD | FUNC_LCM | FUNC_MAX | FUNC_MIN;
 
-func_operator_names_single_arg:
+func_name_single_arg:
     FUNC_ARSINH_NAME | FUNC_ARCOSH_NAME | FUNC_ARTANH_NAME
     | FUNC_ARCSINH_NAME | FUNC_ARCCOSH_NAME | FUNC_ARCTANH_NAME
-    | FUNC_FLOOR_NAME | FUNC_CEIL_NAME;
+    | FUNC_FLOOR_NAME | FUNC_CEIL_NAME
+    | FUNC_RE_NAME | FUNC_IM_NAME | FUNC_ARG_NAME | FUNC_ABS_NAME;
```

Comment:
In addition to adding the new function names here, I did some re-naming and organization of the func rules, similar to what I did in the experimental unknown functions branch